src/features/forest/components/Tree.tsx

```diff
@@ -0,0 +1,221 @@
+import React, { useContext, useRef, useState } from "react";
+import Spritesheet from "react-responsive-spritesheet";
+
+import shakeSheet from "assets/resources/tree/shake_sheet.png";
+import choppedSheet from "assets/resources/tree/chopped_sheet.png";
+import stump from "assets/resources/tree/stump.png";
+import wood from "assets/resources/wood.png";
+import axe from "assets/tools/axe.png";
+
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+import { Context } from "features/game/GameProvider";
+import classNames from "classnames";
+import { useActor } from "@xstate/react";
+import {
+  canChop,
+  CHOP_ERRORS,
+  TREE_RECOVERY_SECONDS,
+} from "features/game/events/chop";
+import { getTimeLeft } from "lib/utils/time";
+import { ProgressBar } from "components/ui/ProgressBar";
+
+const POPOVER_TIME_MS = 1000;
+
+interface Props {
+  treeIndex: number;
+}
+export const Tree: React.FC<Props> = ({ treeIndex }) => {
+  const { gameService, selectedItem } = useContext(Context);
+  const [game] = useActor(gameService);
+
+  const [showPopover, setShowPopover] = useState(true);
```

Comment:
We can refactor this to a state machine. I think it would be much cleaner. Happy to do it though once we have the forest sorted.